msgid ""
msgstr ""
"Project-Id-Version: warpinator\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-11-29 10:55+0000\n"
"PO-Revision-Date: 2023-04-01 11:55+0200\n"
"Last-Translator: László Viktor <szemcse1@gmail.com>\n"
"Language-Team: \n"
"Language: hu_HU\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2022-12-16 10:41+0000\n"
"X-Generator: Poedit 3.2.2\n"

#: resources/group-code.ui:16 src/prefs.py:259
msgid "Group Code"
msgstr "Csoportkód"

#: resources/group-code.ui:39
msgid "You cannot communicate with computers that do not use the same code."
msgstr ""
"Nem kommunikálhat olyan számítógépekkel, amelyek nem ugyanazt a kódot "
"használják."

#: resources/group-code.ui:49 resources/main-window.ui:51
msgid "Set code"
msgstr "Kód beállítása"

#: resources/group-code.ui:93
msgid "Secure mode:"
msgstr "Titkosított mód:"

#: resources/group-code.ui:106 src/prefs.py:594
msgid "OFF"
msgstr "KI"

#: resources/group-code.ui:148
msgid "More information"
msgstr "További információ"

#: resources/main-window.ui:80
msgid "Secure Mode disabled"
msgstr "Titkosított mód kikapcsolva"

#: resources/main-window.ui:263
msgid "No matching results"
msgstr "Nincs eredmény"

#: resources/main-window.ui:419
msgid "Make this remote machine a favorite in the main view"
msgstr "A távoli számítógép kedvencnek jelölése a fő nézetben"

#: resources/main-window.ui:493
msgid "Clear inactive and completed operations"
msgstr "Inaktív és befejezett műveletek törlése"

#: resources/main-window.ui:512
msgid "Send files"
msgstr "Fájlok küldése"

#: resources/main-window.ui:538 src/prefs.py:230
msgid "File Transfers"
msgstr "Fájlátvitel"

#: resources/main-window.ui:559 src/warpinator.py:922
msgid "Online"
msgstr "Elérhető"

#: resources/main-window.ui:679
msgid "No other computers found..."
msgstr "Nem találhatóak további számítógépek…"

#: resources/main-window.ui:744
msgid "Make sure your network connection is active."
msgstr "Bizonyosodjunk meg arról, hogy van elérhető hálózati kapcsolat."

#: resources/main-window.ui:759
msgid ""
"Check your firewall rules.  The port numbers can be changed.  See "
"preferences for more details."
msgstr ""
"Ellenőrizzük a tűzfalszabályokat.  A portszám megváltoztatható.  További "
"részleteket a beállítások között találhatóak."

#. Do not translate 'Warpinator'
#: resources/main-window.ui:775
msgid ""
"Is Warpinator running on any other computers on this network?  If so, make "
"sure they are all using the same group code in preferences."
msgstr ""
"A másik hálózati gépen fut a Warpinator?  Ha igen, győződjünk meg arról, "
"hogy azonos-e a csoport kódja a beállításokban."

#: resources/main-window.ui:793
msgid "Possible solutions:"
msgstr "Lehetséges megoldások:"

#. Do not translate 'Warpinator'
#: resources/main-window.ui:828
msgid "Warpinator is also available on:"
msgstr "A Warpinator elérhető még a következő platformokon is:"

#: resources/main-window.ui:843
msgid "Android"
msgstr "Android"

#: resources/main-window.ui:859
msgid "iOS"
msgstr "iOS"

#: resources/main-window.ui:875
msgid "Windows"
msgstr "Windows (Winpinator)"

#: resources/main-window.ui:949
msgid "Initializing, please wait..."
msgstr "Előkészítés, kis türelmet."

#: resources/main-window.ui:1007
msgid "Something went wrong"
msgstr "Hoppá, valami elromlott"

#: resources/main-window.ui:1082
msgid "Invalid save location"
msgstr "Érvénytelen mentési mappa"

#: resources/main-window.ui:1119
msgid ""
"This folder is inaccessible or does not exist.  Please select a new location "
"in Preferences or ensure that the current one is mounted."
msgstr ""
"Ez a mappa nem hozzáférhető vagy nem létezik.  Kérjük válasszunk egy új "
"helyet a beállításokban vagy győződjünk meg arról, hogy a jelenlegi csatolva "
"van."

#: resources/main-window.ui:1175
msgid "Searching for other computers"
msgstr "További számítógépek keresése"

#: resources/main-window.ui:1234
msgid "Shutting down..."
msgstr "Leállítás…"

#: resources/main-window.ui:1277 src/util.py:439
#: data/org.x.Warpinator.desktop.in.in:5
#: data/org.x.Warpinator.appdata.xml.in.in:7
msgid "Send and Receive Files across the Network"
msgstr "Fájlok küldése és fogadása a hálózaton"

#: resources/op-item.ui:140
msgid "Awaiting approval"
msgstr "Jóváhagyásra várva"

#: resources/op-item.ui:156
msgid "Not enough disk space!"
msgstr "Nincs elegendő lemezterület!"

#: resources/op-item.ui:219
msgid "Calculating"
msgstr "Számítás"

#: resources/op-item.ui:259
msgid "Accept this request"
msgstr "Kérés elfogadása"

#: resources/op-item.ui:274
msgid "Refuse this request"
msgstr "Kérés visszautasítása"

#: resources/op-item.ui:289
msgid "Cancel this request"
msgstr "Kérés megszakítása"

#: resources/op-item.ui:304
msgid "Restart this operation"
msgstr "Művelet újraindítása"

#: resources/op-item.ui:319
msgid "Pause this operation"
msgstr "Művelet szüneteltetése"

#: resources/op-item.ui:334
msgid "Resume this operation"
msgstr "Művelet folytatása"

#: resources/op-item.ui:349
msgid "Cancel this operation"
msgstr "Művelet megszakítása"

#: resources/op-item.ui:364
msgid "Remove this operation from the list"
msgstr "Művelet eltávolítása a listából"

#: resources/op-item.ui:379
msgid "Open the containing folder"
msgstr "Tartalmazó mappa megnyitása"

#: resources/prefs-window.ui:9
msgid "Note on port numbers:"
msgstr "Megjegyzés a portok számával kapcsolatban:"

#. Do not translate 'Warpinator'
#: resources/prefs-window.ui:24
msgctxt "preferences window title"
msgid "Warpinator Preferences"
msgstr "Beállítások"

#: src/notifications.py:23
msgid "New incoming files"
msgstr "Új beérkező fájlok"

#: src/notifications.py:29
#, python-format
msgid ""
"%(sender_name)s would like to send you %(file_count)d file (%(file_name)s)."
msgstr ""
"%(sender_name)s szeretne Önnek %(file_count)d fájlt küldeni (%(file_name)s)."

#: src/notifications.py:30
#, python-format
msgid "%(sender_name)s would like to send you %(file_count)d files"
msgstr "%(sender_name)s szeretne Önnek %(file_count)d fájlt küldeni"

#: src/notifications.py:42
msgid "Accept"
msgstr "Elfogadás"

#: src/notifications.py:43
msgid "Decline"
msgstr "Elutasítás"

#: src/notifications.py:57
#, python-format
msgid "%(sender_name)s is sending you %(file_count)d file (%(file_name)s)."
msgstr "%(sender_name)s %(file_count)d fájlt küld Önnek (%(file_name)s)."

#: src/notifications.py:58
#, python-format
msgid "%(sender_name)s is sending you %(file_count)d files"
msgstr "%(sender_name)s %(file_count)d fájlt küld Önnek"

#: src/notifications.py:94
msgid "Transfer complete"
msgstr "Az átvitel sikeresen befejeződött"

#: src/notifications.py:97
#, python-format
msgid "The transfer to %s has finished, but with errors"
msgstr "A fájlok átvitele %s részére hibákkal ugyan, de sikeresen befejeződött"

#: src/notifications.py:99
#, python-format
msgid "The transfer to %s has finished successfully"
msgstr "A fájlok átvitele %s részére sikeresen befejeződött"

#: src/notifications.py:102
#, python-format
msgid "The transfer from %s has finished, but with errors"
msgstr ""
"A fájlok átvitele %s részéről hibákkal ugyan, de sikeresen befejeződött"

#: src/notifications.py:104
#, python-format
msgid "The transfer from %s has finished successfully"
msgstr "A fájlok átvitele %s részéről sikeresen befejeződött"

#: src/notifications.py:138 src/warpinator.py:175 src/warpinator.py:178
msgid "Transfer failed"
msgstr "Sikertelen átvitel"

#: src/notifications.py:141
#, python-format
msgid "Something went wrong with the transfer to %s"
msgstr "Hiba történt a fájlok %s részére történő átvitele során"

#: src/notifications.py:143
#, python-format
msgid "Something went wrong with the transfer from %s"
msgstr "Hiba történt a fájlok %s részéről történő átvitele során"

#: src/notifications.py:172 src/warpinator.py:169
msgid "Transfer cancelled"
msgstr "Átvitel megszakítva"

#: src/notifications.py:175
#, python-format
msgid "Your transfer to %s was cancelled"
msgstr "%s részére történő küldés megszakítva"

#: src/notifications.py:177
#, python-format
msgid "An incoming transfer from %s was cancelled"
msgstr "%s részéről történő átvitel megszakítva"

#: src/ops.py:49
#, python-format
msgid "%(time_left)s (%(bytes_per_sec)s/s)"
msgstr "%(time_left)s (%(bytes_per_sec)s/s)"

#. Translators: Don't need to translate singular, we show the filename if there's only one
#: src/ops.py:150
#, python-format
msgid "%d file-do-not-translate"
msgid_plural "%d files"
msgstr[0] "%d fájl"
msgstr[1] "%d fájl"

#. Translators: Don't need to translate singular, we show the filename if there's only one
#: src/ops.py:238
#, python-format
msgid "%d file"
msgid_plural "%d files"
msgstr[0] "%d fájl"
msgstr[1] "%d fájl"

#: src/prefs.py:196
msgid "Warpinator Preferences"
msgstr "Beállítások"

#: src/prefs.py:212
msgid "General"
msgstr "Általános"

#: src/prefs.py:214
msgid "Desktop"
msgstr "Asztal"

#: src/prefs.py:216
msgid "Show an icon in the notification area"
msgstr "Ikon megjelenítése az értesítési területen"

#: src/prefs.py:220
msgid "Start with main window open"
msgstr "Indulás a főablak megnyitásával"

#: src/prefs.py:225
msgid "Start automatically"
msgstr "Automatikus indítás"

#: src/prefs.py:232
msgid "Use compression when possible."
msgstr "Tömörítés használata, amennyiben lehetséges."

#: src/prefs.py:234
msgid ""
"Warning: This may have a negative impact on performance on some faster "
"networks."
msgstr ""
"Figyelmeztetés: Ez a beállítás gyorsabb hálózatokon ronthatja a "
"teljesítményt."

#: src/prefs.py:237
msgid "Location for received files"
msgstr "Fogadott fájlok mentési mappája"

#: src/prefs.py:242
msgid "Require approval before accepting files"
msgstr "Fogadott fájlok jóváhagyása szükséges"

#: src/prefs.py:247
msgid "Require approval when files would be overwritten"
msgstr "Fájlok felülírásához jóváhagyása szükséges"

#: src/prefs.py:252
msgid "Display a notification when someone sends (or tries to send) you files"
msgstr "Értesítés megjelenítése, ha valaki fájlokat küld (vagy küldeni próbál)"

#: src/prefs.py:257
msgid "Connection"
msgstr "Kapcsolat"

#: src/prefs.py:267
msgid "Network"
msgstr "Hálózat"

#: src/prefs.py:270
msgid "Automatic"
msgstr "Automatikus"

#. translation: combobox item shown when a previosuly set interface can no longer be found - 'wlan0 - not found'
#: src/prefs.py:349
#, python-format
msgid "%s - not found"
msgstr "%s nem található"

#: src/prefs.py:351
msgid "Network interface to use"
msgstr "Használni kívánt hálózati csatoló"

#: src/prefs.py:359
msgid "Incoming port for transfers"
msgstr "Beérkező átvitel forgalmának portja"

#: src/prefs.py:366
msgid "Incoming port for registration"
msgstr "Beérkező regisztráció forgalmának portja"

#: src/prefs.py:381
msgid "Apply network changes"
msgstr "Hálózati változtatások alkalmazása"

#: src/prefs.py:390
msgid "Update firewall rules"
msgstr "Tűzfal szabályok frissítése"

#: src/prefs.py:402
msgid ""
"Any port numbers will work for the application, but using the same ports for "
"all computers can make it simpler to add firewall exceptions if necessary."
msgstr ""
"Bármilyen nem foglalt portszám használható az alkalmazás számára. Ugyanazt a "
"portot használva az összes gépen egyszerűbbé tehető a tűzfalszabály "
"beállítása, amennyiben ez szükséges."

#: src/prefs.py:475
msgid "Only available in secure mode."
msgstr "Csak titkosított módban érhető el."

#: src/prefs.py:552
msgid "A group code is required."
msgstr "A csoportkód megadása szükséges."

#: src/prefs.py:557
msgid "The group code is too short."
msgstr "A csoportkód túl rövid."

#: src/prefs.py:591
msgid "ON"
msgstr "BE"

#: src/prefs.py:592
msgid "All options are unlocked."
msgstr "Minden opció fel van oldva."

#: src/prefs.py:595
msgid ""
"You must set a custom group code to run in secure mode. Some options are "
"currently disabled."
msgstr ""
"Be kell állítania egy egyéni csoportkódot ahhoz, hogy titkosított módban "
"működjön. Néhány beállítás jelenleg le van tiltva."

#: src/remote.py:490
#, python-format
msgid ""
"Transfer completed, but the number of files received is less than the "
"original request size (expected %d, received %d)"
msgstr ""
"Az átvitel befejeződött, de a fogadott fájlok száma kisebb, mint az eredeti "
"kérés mérete (várt %d, érkezett %d)"

#: src/remote.py:613 src/remote.py:617
msgid "Connection has been lost"
msgstr "A kapcsolat elveszett"

#: src/server.py:566
msgid "An error occurred on the remote machine"
msgstr "Hiba lépett fel a távoli számítógépen"

#: src/transfers.py:216
msgid "File count exceeds original request size"
msgstr "A fájlok száma meghaladja az eredeti kérés méretét"

#: src/transfers.py:224
#, python-format
msgid "Resolved path is not valid: %s -> %s"
msgstr "A megoldott elérési út nem érvényes: %s -> %s"

#: src/util.py:153
msgid "Cancel"
msgstr "Mégse"

#: src/util.py:154
msgid "Add"
msgstr "Hozzáadás"

#: src/util.py:176
msgid "Select file(s) to send"
msgstr "Válasszuk ki a küldendő fájl(oka)t"

#: src/util.py:328
msgid "A few seconds remaining"
msgstr "Néhány másodperc múlva befejeződik"

#: src/util.py:331
#, python-format
msgid "%d seconds remaining"
msgstr "%d másodperc van hátra"

#: src/util.py:335 src/util.py:343
#, python-format
msgid "%d minute"
msgid_plural "%d minutes"
msgstr[0] "%d perc"
msgstr[1] "%d perc"

#: src/util.py:342
#, python-format
msgid "%d hour"
msgid_plural "%d hours"
msgstr[0] "%d óra"
msgstr[1] "%d óra"

#: src/util.py:347
#, python-format
msgid "approximately %d hour"
msgid_plural "approximately %d hours"
msgstr[0] "nagyjából %d óra"
msgstr[1] "nagyjából %d óra"

#: src/util.py:435 src/warpinator.py:548
msgid "About"
msgstr "Névjegy"

#: src/warpinator.py:148
msgid "Paused"
msgstr "Szüneteltetve"

#: src/warpinator.py:151
msgid "Waiting for approval"
msgstr "Jóváhagyásra várva"

#: src/warpinator.py:153
msgid "Waiting for your approval"
msgstr "Jóváhagyására vár"

#: src/warpinator.py:156
msgid "Files may be overwritten"
msgstr "A már létező fájlok felül lesznek írva"

#: src/warpinator.py:159
msgid "Not enough disk space"
msgstr "Nincs elegendő lemezterület"

#: src/warpinator.py:162
msgid "Request cancelled"
msgstr "Kérés visszautasítva"

#: src/warpinator.py:164 src/warpinator.py:166
#, python-format
msgid "Request cancelled by %s"
msgstr "A kérést %s visszautasította"

#: src/warpinator.py:171 src/warpinator.py:173
#, python-format
msgid "Transfer cancelled by %s"
msgstr "Az átvitelt %s megszakította"

#: src/warpinator.py:180
#, python-format
msgid "File '%s' not found"
msgstr "'%s' fájl nem található"

#: src/warpinator.py:182
msgid "Some files not found"
msgstr "Nem találhatók fájlok"

#: src/warpinator.py:184
msgid "Completed"
msgstr "Befejezve"

#: src/warpinator.py:186
msgid "Completed, but with errors"
msgstr "Befejezve hibákkal"

#: src/warpinator.py:345 src/warpinator.py:950
msgid "Connecting"
msgstr "Kapcsolódás"

#: src/warpinator.py:347
#, python-format
msgid "Connecting to %s"
msgstr "Kapcsolódás a következőhöz: %s"

#: src/warpinator.py:357
#, python-format
msgid "%s is not online"
msgstr "%s most nem elérhető"

#: src/warpinator.py:362
#, python-format
msgid "Cannot connect to %s"
msgstr "Nem lehet kapcsolódni a következőhöz: %s"

#: src/warpinator.py:367
#, python-format
msgid "Waiting for %s to complete the connection."
msgstr "Várakozás %s partnerre a kapcsolat létrehozásához."

#: src/warpinator.py:396
#, python-format
msgid "%d new incoming transfer"
msgid_plural "%d new incoming transfers"
msgstr[0] "%d beérkező kapcsolat"
msgstr[1] "%d beérkező kapcsolat"

#: src/warpinator.py:513 src/warpinator.py:1385
msgid "Favorites"
msgstr "Kedvencek"

#: src/warpinator.py:518 src/warpinator.py:1390
msgid "Recents"
msgstr "Legutóbbi fájlok"

#: src/warpinator.py:525 src/warpinator.py:1399
msgid "Browse..."
msgstr "Tallózás…"

#: src/warpinator.py:540 src/warpinator.py:1331
msgid "Open save folder"
msgstr "Mentési mappa megnyitása"

#: src/warpinator.py:544
msgid "Preferences"
msgstr "Beállítások"

#: src/warpinator.py:552 src/warpinator.py:1334
msgid "Quit"
msgstr "Kilépés"

#: src/warpinator.py:645
msgid "You don't appear to be connected to a network."
msgstr "Nincs hálózati kapcsolat."

#: src/warpinator.py:647
msgid "Startup was unsuccessful, please check your logs."
msgstr "Az alkalmazás indítása meghiúsult, ellenőrizzük le a hibanaplókat."

#: src/warpinator.py:739
#, python-format
msgid ""
"Warpinator is running with limited functionality, and will exit in %d "
"minute. To enable Secure Mode, set a unique group code."
msgstr ""
"A Warpinator korlátozott funkcionalitással fut, és %d perc múlva kilép. A "
"titkosított mód engedélyezéséhez állítsunk be egy egyedi csoportkódot."

#: src/warpinator.py:741
#, python-format
msgid ""
"Warpinator is running with limited functionality, and will exit in %d "
"minutes. To enable Secure Mode, set a unique group code."
msgstr ""
"A Warpinator korlátozott funkcionalitással fut, és %d perc múlva kilép. A "
"titkosított mód engedélyezéséhez állítsunk be egy egyedi csoportkódot."

#: src/warpinator.py:929
msgid "Offline"
msgstr "Nem érhető el"

#: src/warpinator.py:936
msgid "Unable to connect"
msgstr "Nem lehet kapcsolódni"

#: src/warpinator.py:943
msgid "Waiting for two-way connection"
msgstr "Várakozás a kétirányú kapcsolatra"

#: src/warpinator.py:1368
msgid "Send to all"
msgstr "Küldés mindenkinek"

#: data/org.x.warpinator.policy.in.in:12
msgid "Open a firewall port for Warpinator"
msgstr "Tűzfalport nyitása a Warpinator számára"

#: data/org.x.warpinator.policy.in.in:13
msgid "To modify firewall settings, you need to authenticate."
msgstr "A tűzfalszabályok módosításához meg kell adnia a jelszavát."

#. DO NOT TRANSLATE Warpinator
#: data/org.x.Warpinator.appdata.xml.in.in:6
msgid "Warpinator"
msgstr "Warpinator"

#: data/org.x.Warpinator.appdata.xml.in.in:9
msgid ""
"Warpinator allows you to easily connect multiple computers on a local area "
"network and share files quickly and securely."
msgstr ""
"A Warpinator segítségével könnyedén kapcsolhat össze számítógépeket a helyi "
"hálózaton és oszthat meg gyorsan és biztonságosan fájlokat."

#. DO NOT TRANSLATE Linux Mint
#: data/org.x.Warpinator.appdata.xml.in.in:27
msgid "Linux Mint"
msgstr "Linux Mint"
